### PR TITLE
Quickstart based integration test [VS-357]

### PR DESCRIPTION
**Pull request by mcovarr**
Fully manual for now, also turns off filtering as that appears to be nondeterministic and this currently only does exact matching of actual to expected results.

### CODE REVIEW

File: scripts/variantstore/wdl/GvsQuickstartIntegration.wdl [hunk 1]
```diff
@@ -0,0 +1,224 @@
...
+    call GvsUnified.GvsUnified {
+        input:
+            dataset_name = Prepare.dataset_name,
+            project_id = "spec-ops-aou",
+            external_sample_names = external_sample_names,
+            gatk_override = Prepare.jar,
+            input_vcfs = input_vcfs,
+            input_vcf_indexes = input_vcf_indexes,
+            filter_set_name = "quickit",
+            create_filter_set_scatter_count = 20,
```
Comment (RoriCremer) [inline scripts/variantstore/wdl/GvsQuickstartIntegration.wdl:65]: do we still need these filter params if we are turning filtering off for now (extract_do_not_filter_override) ? (The answer may totally be yes and someplace else needs that updated)

File: scripts/variantstore/wdl/GvsQuickstartIntegration.wdl [hunk 2]
```diff
@@ -0,0 +1,224 @@
...
+            fail=1
+          fi
+        done
+
+        if [[ $fail -ne 0 ]]; then
+          exit 1
+        fi
+
+        echo "All vcfs compared and matched!"
+    >>>
```
Comment (RoriCremer) [inline scripts/variantstore/wdl/GvsQuickstartIntegration.wdl:213]: When we are ready to start doing fuzzy comparisons (with filtering), we could potentially use the ah_var_store docker image which has bcftools in it and do some easy selective comparisons that way. eg take two vcfs and compare only their variants as opposed to their bytes

Comment (RoriCremer) [reply scripts/variantstore/wdl/GvsQuickstartIntegration.wdl:213]: bcftools would also allow us to merge all of the shards into one fairly quickly which could also speed up the comparison

File: scripts/variantstore/wdl/GvsQuickstartIntegration.wdl [hunk 3]
```diff
@@ -0,0 +1,224 @@
...
+    output {
+        Array[File] output_vcfs = GvsUnified.output_vcfs
+        Array[File] output_vcf_indexes = GvsUnified.output_vcf_indexes
+        Float total_vcfs_size_mb = GvsUnified.total_vcfs_size_mb
+        File manifest = GvsUnified.manifest
+        Boolean done = true
+    }
+}
+
+task Prepare {
```
Comment (rsasch) [inline scripts/variantstore/wdl/GvsQuickstartIntegration.wdl:88]: nit: can this be something else, like "setup", just to not create any confusion with the `PrepareRangesCallset` step?

**Review by rsasch — APPROVED** [review]
one small nit, otherwise 👍🏻

**Review by gbggrant — APPROVED** [review]

**Pull request merged**